.travis.yml

```diff
@@ -65,6 +65,16 @@ install:
       fi
     fi
 
+# JUST FOR NOW : Install latest master version of iris-grib.
```

Comment:
Aren't we going to have to do this every time we change the Iris API in a way that breaks the currently released iris-grib? Additionally, when that happens we need to make sure we fix iris-grib at the same time to keep the tests passing. I can't help thinking we need to break the circular dependency. That would mean no Iris tests which use iris-grib

Reply:
Alternatively: Make Iris-grib no longer depend on Iris somehow.

Reply:
> every time we change the Iris API in a way that breaks the currently released iris-grib?

Well yes, but that shouldn't really be any more frequent than breaking changes to user code.
That is, if Iris-Grib only depended on Iris ***public*** api.
That should be perfectly practical -- and right now is *nearly* true -- but unfortunately still a way off.

> no Iris tests which use iris-grib

I'm working on that = main reason why this PR is not yet ready.
I think we can + will still have tests that use iris-grib in an integration-type way, but we will remove anything depending on iris-grib internals.  Again it's WIP.

> Make Iris-grib no longer depend on Iris somehow.

Really not practical I think.  But we certainly can ensure that Iris does not depend on Iris-grib.
At this point, the hard part is still the tests :  
 * some tests refer to internals of iris-grib, or gribapi :  these should either be ditched or moved to iris-grib.  
 * all tests *in* iris-grib use the Iris testing mechanics : need to ensure that is only public api
 * some Iris testing code is grib-specific : need to move to iris-grib

Reply:
>   no Iris tests which use iris-grib ...  = main reason why this PR is not yet ready.

Correction : that would be nice, but is not required for Iris 2.0 release, so I'm shelving it for now.
See [iris-grib#88](https://github.com/SciTools/iris-grib/issues/88)